src/pages/form/advanced-form/index.tsx

```diff
@@ -0,0 +1,308 @@
+                      <Select placeholder="请选择审批员">
+                        <Option value="xiao">付晓晓</Option>
+                        <Option value="mao">周毛毛</Option>
+                      </Select>,
+                    )}
+                  </Form.Item>
+                </Col>
+                <Col xl={{ span: 6, offset: 2 }} lg={{ span: 8 }} md={{ span: 12 }} sm={24}>
+                  <Form.Item label={fieldLabels.dateRange}>
+                    {getFieldDecorator('dateRange', {
+                      rules: [{ required: true, message: '请选择生效日期' }],
+                    })(
+                      <RangePicker
+                        placeholder={['开始日期', '结束日期']}
+                        style={{ width: '100%' }}
+                      />,
+                    )}
+                  </Form.Item>
+                </Col>
+                <Col xl={{ span: 8, offset: 2 }} lg={{ span: 10 }} md={{ span: 24 }} sm={24}>
+                  <Form.Item label={fieldLabels.type}>
+                    {getFieldDecorator('type', {
+                      rules: [{ required: true, message: '请选择仓库类型' }],
+                    })(
+                      <Select placeholder="请选择仓库类型">
+                        <Option value="private">私密</Option>
+                        <Option value="public">公开</Option>
+                      </Select>,
+                    )}
+                  </Form.Item>
+                </Col>
+              </Row>
+            </Form>
+          </Card>
+          <Card title="任务管理" className={styles.card} bordered={false}>
+            <Form layout="vertical" hideRequiredMark>
+              <Row gutter={16}>
+                <Col lg={6} md={12} sm={24}>
+                  <Form.Item label={fieldLabels.name2}>
+                    {getFieldDecorator('name2', {
+                      rules: [{ required: true, message: '请输入' }],
+                    })(<Input placeholder="请输入" />)}
+                  </Form.Item>
+                </Col>
+                <Col xl={{ span: 6, offset: 2 }} lg={{ span: 8 }} md={{ span: 12 }} sm={24}>
+                  <Form.Item label={fieldLabels.url2}>
+                    {getFieldDecorator('url2', {
+                      rules: [{ required: true, message: '请选择' }],
+                    })(<Input placeholder="请输入" />)}
+                  </Form.Item>
+                </Col>
+                <Col xl={{ span: 8, offset: 2 }} lg={{ span: 10 }} md={{ span: 24 }} sm={24}>
+                  <Form.Item label={fieldLabels.owner2}>
+                    {getFieldDecorator('owner2', {
+                      rules: [{ required: true, message: '请选择管理员' }],
+                    })(
+                      <Select placeholder="请选择管理员">
+                        <Option value="xiao">付晓晓</Option>
+                        <Option value="mao">周毛毛</Option>
+                      </Select>,
+                    )}
+                  </Form.Item>
+                </Col>
+              </Row>
+              <Row gutter={16}>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 16](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)